packages/demand-bloc-quotes/src/transformer.ts

```diff
@@ -0,0 +1,58 @@
+import { QuoteItem as OriginalQuoteItem } from '@karhoo/demand-api'
+
+export interface QuoteItem {
+  currencyCode: string
+  eta?: number | null
+
+  finalPrice?: null
+  fleetId: string
+  fleetLogo: string
+  fleetName: string
+  fleetPhoneNumber: string
+  fleetTermsAndConditions: string
+  id: string
+  quotedPrice: number | null
+  type: 'FIXED' | 'ESTIMATED' | 'METERED' | ''
```

Comment:
You can use enum `QuotePriceTypes` which you defined in types

Reply:
yep, will do it once we publish a new version with this changes